renderdoc/driver/vulkan/vk_manager.cpp

```diff
@@ -478,6 +500,28 @@ void VulkanResourceManager::InsertDeviceMemoryRefs(WriteSerialiser &ser)
   }
 }
 
+void VulkanResourceManager::InsertImageRefs(WriteSerialiser &ser)
+{
+  std::vector<ImgRefsPair> data;
+  data.reserve(m_ImgFrameRefs.size());
+  uint32_t sizeEstimate = 32;
+
+  for(auto it = m_ImgFrameRefs.begin(); it != m_ImgFrameRefs.end(); it++)
+  {
+    data.push_back({it->first, it->second});
+    sizeEstimate += 8 * (1 +                                        // rangeRefs Size
+                         (uint32_t)it->second.rangeRefs.size() +    // rangeRefs elements
+                         8 +                                        // imageInfo
+                         1 +                                        // aspectMask
+                         3);                                        // are*Split
```

Comment:
The size estimate stuff is always admittedly a bit nasty, but do you think it would be reasonable to simplify this to `sizeof(ImgRefsPair) + sizeof(FrameRefType) * it->second.rangeRefs.size()` here?

That would be my first instinct, since the serialised form is generally always lower than `sizeof()` due to lack of padding, with only dynamic elements like vectors needing special treatment.

Reply:
Ah, I see. I was under the mistaken impression that these fields would be encoded as 64-bit values. I will change the calculation along with the CI fix.